boards/arm/nucleo_l433rc_p/nucleo_l433rc_p.dts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2019 IP-Logix Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/l4/stm32l433Xc.dtsi>
+#include <st/l4/stm32l433r(b-c)tx-pinctrl.dtsi>
+#include "arduino_r3_connector.dtsi"
+
+/ {
+	model = "STMicroelectronics STM32L433RC-P NUCLEO board";
+	compatible = "st,stm32l433rc-nucleo-p", "st,stm32l433";
```

Comment:
Please remove the soc compatible:
```suggestion
	compatible = "st,stm32l433rc-nucleo-p";
```

Reply:
I get the `DT compatible string "st,stm32l433rc-nucleo-p" appears un-documented` warning from checkpatch.

Reply:
@arvinf when running checkpatch locally ? Which is strange is that it is not reported in github by zephyr-bot.
Anyway `compatible = "st,stm32l433rc-nucleo-p";` is the correct way and there is no reason to complain as "st" is documented in dts/bindings/vendor-prefixes.txt (maybe checkpatch doesn't find it)

Reply:
Yes. Locally. I checked `dts/bindings/vendor-prefixes.txt` and you are correct, `st` is listed.

```
DT compatible string "st,stm32l433rc-nucleo-p" appears un-documented -- check ./dts/bindings/

#52: FILE: boards/arm/nucleo_l433rc_p/nucleo_l433rc_p.dts:14:
+	compatible = "st,stm32l433rc-nucleo-p";

- total: 0 errors, 1 warnings, 55 lines checked

```

In any event, I made the changes.

Reply:
@galak do you why checkpatch reports this kind of error locally and not in github action ? Not a real issue, but  it would nice to understand what happens